lib/services/ngsi/entities-NGSI-LD.js

```diff
@@ -68,93 +68,98 @@ function valueOfOrNull(value) {
 
```

Comment:
I understand this PR doesn't need to add any entry to CNR file given the existing "Add: basic NGSI-LD support as experimental feature (#842)" already covers it, right?

Reply:
Correct.

Reply:
NTC